src/core/msg_translator.c

```diff
@@ -2845,6 +2876,47 @@ int branch_builder(unsigned int hash_index,
 	return size;
 }
 
+static int is_haproxy(struct dest_info *send_info, struct receive_info *haproxy_rcv) {
```

Comment:
Can you add some comment to describe the purpose of the function? The name is rather generic, from C code point of view it looks like it tries to figure out if the target via via haproxy...